webapi/common.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import unicode_literals
```

Comment:
Here and in other files: Deluge is now Py 3 only, do future imports are of no use.

Reply:
Will do.